EventListener/Login.php

```diff
@@ -63,6 +67,21 @@ public function setCustomerId($value)
         $this->options['customerId'] = $value;
     }
 
+    protected function getUserIdentifier(TokenInterface $token)
+    {
+        $tokenUser = $token->getUser();
+
+        if ($tokenUser instanceof eZUser || !$tokenUser instanceof UserInterface) {
```

Comment:
Hi @Plopix, good work and thank you.
I'm not sure about the second condition. Can you explain this? `|| !$tokenUser instanceof UserInterface`